srcalc/src/main/java/gov/va/med/srcalc/domain/model/ValueMatcher.java

```diff
@@ -96,9 +101,62 @@ public Expression getParsedExpression()
         return fBooleanExpression;
     }
     
+    /**
+     * Returns true if this matcher should be evaluated and false if it should not
+     * be evaluated.
+     */
+    @Basic
+    public boolean isEnabled()
+    {
+        return fEnabled;
+    }
+    
+    void setEnabled(final boolean enabled)
+    {
+        fEnabled = enabled;
+    }
+    
+    /**
+     * Evaluate this ValueMatcher's boolean expression based on the given context and value.
```

Comment:
This doc comment should specify the behavior if the "apply condition" flag is false.